scripts/biomedical/NCBI_Taxonomy/test_data/division_enum_expected.mcf

```diff
@@ -0,0 +1,78 @@
+Node:dcid:BiologicalTaxonomicDivisionEnum
+name: "BiologicalTaxonomicDivisionEnum"
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "These are broad biological divisions that are commonly used to divide living organisms"
+
+Node:dcid:BiologicalTaxonomicDivisionBacteria
+name: Bacteria
+typeOf: dcs:BiologicalTaxonomicDivisionEnum
+acronym: BCT
+description:  
```

Comment:
If a description value is empty, then please don't write the property with no value - please update this in your script. This is throwing trailing whitespace errors.